ui/src/shared/components/Dygraph.tsx

```diff
@@ -459,8 +459,11 @@ class Dygraph extends Component<Props, State> {
   }
 
   private getLabel = (axis: string): string => {
-    const {axes, queries} = this.props
-    const label = getDeep<string>(axes, `${axis}.label`, '')
+    const {axes, labels, queries} = this.props
+
+    // if label comes back as '', use the y axis label from props.labels
+    // see https://github.com/influxdata/chronograf/issues/5314
+    const label = getDeep<string>(axes, `${axis}.label`, '') || labels[1]
```

Comment:
How can we be certain that `label` will have a length greater than 0?

Reply:
@Asalem1 fixed this